general-concepts/use-flags/text.xml

```diff
@@ -194,6 +194,38 @@ To get a flag added to <c>use.mask</c>, ask the relevant arch team.
 </body>
 </section>
 
+<section>
+<title>IUSE defaults</title>
+<body>
+
+<p>
+Add <c>+</c> or <c>-</c> before the name of the use flag in <c>IUSE</c> to turn
+it on or off by default.
+</p>
+
+<important>
+The default USE-ordering is
+<c>USE_ORDER="env:pkg:conf:defaults:pkginternal:features:repo:env.d"</c>
+(see man make.conf).
+</important>
+
+<important>
+Disabling default <c>IUSE</c> is pretty much useless as it does not override
```

Comment:
I know you just moved it but I really have no clue what this means.

Reply:
It means that `pkginternal` has lower precedence than `conf` and `defaults`.
I'll try to explain it in an understandable way.